encrypted-media/clearkey-mp4-playback-persistent-license-events.html

```diff
@@ -0,0 +1,52 @@
+<!doctype html>
+<html>
+  <head>
+    <meta charset=utf-8>
+    <meta name="timeout" content="long">
+    <title>Encrypted Media Extensions: Successful Playback, persistent-usage-record session with Clear Key, mp4, event sequence</title>
```

Comment:
"persistent-usage-record" should be "persistent-license" three places in this PR.